pom.xml

```diff
@@ -1241,6 +1243,7 @@
         <module>tensorflow</module>
         <module>ale</module>
         <module>onnx</module>
+        <module>ngraph</module>
```

Comment:
Let's remove it from the list for Mac here, unless it's supposed to work already?